src/plays/inspirational-quotes/Readme.md

```diff
@@ -0,0 +1,27 @@
+# InspirationalQuotes
+
+This app contains motivational quotes. This app has over 250 carefully chosen quotes to motivate users. I did not produce quotations using any API. I have hand-picked these relevant quotes.
+
+## Play Demographic
+
+- Language: ts
+- Level: Beginer
+
+## Creator Information
+
+- User: Muzaffar Hossain
+- Gihub Link: https://github.com/muzaffar640
+- Blog:
+- Video:
+
+## Implementation Details
+
+This implementation uses Typescript and Hooks.
```

Comment:
@muzaffar640 Can you please add some implementation details? Like What do you do fetch the quotes, how you manage states etc? As there is no content like blog and video associated, this read will be helpful to anyone learning from your code.

Reply:
Added some steps of implementation. Please have a look.